dev-lang/yab/yab-1.8.1.recipe

```diff
@@ -5,22 +5,23 @@ HOMEPAGE="https://software.besly.de/"
 COPYRIGHT="1995-2006 Marc-Oliver Ihm (yabasic)
 	2006-2009 Jan Bungeroth (yab)
 	2013-2020 Jim Saxton (yab improvements)
-	2018-2021 Lorenz Glaser (yab improvements)"
+	2018-2022 Lorenz Glaser (yab improvements)"
 LICENSE="Artistic"
-REVISION="3"
+REVISION="1"
 SOURCE_URI="https://github.com/lorglas/yab/archive/$portVersion.tar.gz"
-CHECKSUM_SHA256="947560212c2ab7b56da4ebd9b732afc6c4839adc7667c7ab13cfd2ef5c982a76"
+CHECKSUM_SHA256="abeeb6d5d525c8b06f28c78bec52bcac6688967ae27c612a9a13cbbe4b17c933"
 SOURCE_FILENAME="yab-$portVersion.tar.gz"
 SOURCE_DIR="yab-$portVersion"
 
-ARCHITECTURES="all !x86"
+ARCHITECTURES="any!x86"
 
 PROVIDES="
 	yab = $portVersion
 	cmd:yab = $portVersion
 	lib:libyab = $portVersion
 	lib:libyab_1.7.8 = 1.7.8
 	lib:libyab_1.7.9 = 1.7.9
+	lib:libyab_1.8.0 = 1.8.0
```

Comment:
actually these three old libs are only available on x86_gcc2 and x86_64

Reply:
@lorglas are these "old" libraries still needed?